### PR TITLE
PR for Issue 21499: Update jwkEnabled description for OIDC OP

### LINKED ISSUE #21499: Update the description for the jwkEnabled attribute in openidConnectProvider
The description for jwkEnabled attribute in openidConnectProvider needs to be updated to specify that,

When JWK is enabled, the openidConnectProvider dynamically generates key pairs and signs the JWT token with the private key. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk. When this attribute is false, the client can still retrieve the key, specified by keyAliasName, from the JWK endpoint as a JWK.

### PR DESCRIPTION
**Pull request by ayoho**
Resolves #21499

### CODE REVIEW

**Comment by ayoho** [comment]
#build

**Comment by posmith** [comment]
L2 message review not required

**Review by barbj — CHANGES_REQUESTED** [review]
I don't understand this sentence:
When this attribute is false, the client can still retrieve the key specified by the keyAliasName attribute as a JWK from the JWK endpoint.

**Comment by ayoho** [comment]
#libby

File: dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -171,8 +171,8 @@ customClaims.desc=The extra claims to be put in the payloads of the ID token, in
 thirdPartyIDTokenClaims=Extra claims from a third-party ID token
 thirdPartyIDTokenClaims.desc=The extra claims from a third-party ID token to be put in the payload of the ID token.
 
-jwkEnabled=Enable JSON web key(JWK)
-jwkEnabled.desc=Enables or disables JWK.
+jwkEnabled=Enable JSON web key (JWK)
+jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk. When this attribute is false, the provider will use the key that is specified by the keyAliasName attribute to sign the JWT. The provider will still make that key available as a JWK from the JWK endpoint if this attribute is false.
```
Comment (hiltoncv) [inline dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties:175]: Proposed text:
"jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk.
When this attribute is false, the provider will use the key that is specified by the keyAliasName attribute to sign the JWT. The provider will still make that key available as a JWK from the JWK endpoint if this attribute is false."

Comment (ayoho) [reply dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties:175]: @hiltoncv I don't see any difference between your proposed text and what I have.

Comment (barbj) [reply dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties:175]: @hiltoncv All the words appear the same to me also.  The only difference I see is line breaks.

**Review by barbj — APPROVED** [review]

File: dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (hiltoncv) [reply dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties:175]: With the latest iteration, the proposed text is not different from the original text. The original text is good to go.

Comment (hiltoncv) [reply dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties:175]: > Proposed text:
> "jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://<host_name>:<port_number>/oidc/endpoint/<provider_id>/jwk.
> When this attribute is false, the provider will use the key that is specified by the keyAliasName attribute to sign the JWT. The provider will still make that key available as a JWK from the JWK endpoint if this attribute is false."

Disregard the proposed text. The original text can be retained.

**Pull request merged**